src/main/kotlin/no/nav/syfo/api/model/VurderingResponseDTO.kt

```diff
@@ -38,12 +38,14 @@ data class VarselDTO private constructor(
     val uuid: UUID,
     val createdAt: LocalDateTime,
     val svarfrist: LocalDate,
+    val isExpired: Boolean,
```

Comment:
Legger til i DTO slik at vi kan bruke den i stedet for å ha duplisert logikk for utløpt i frontend.